### ISSUE TITLE
Runnable - Scan all network interfaces with an ignore list

### ISSUE THREAD

**Issue by liutingdu** [evercam/evercam-discovery-java#17]

**Comment by scandox** [comment]
@liutingdu Agree that the simplest way to move forward is to allow the runnable library to accept a range option. That way the Gateway can call the library repeatedly setting different ranges. While this is not a beautiful design it has the merit of being extremely easy.

**Comment by liutingdu** [comment]
@scandox 

I've added the IP address and subnet mask option for the runnable jar, now you can pass the values by using: ```java -jar evercam-discovery.jar -ip 192.168.0.90 -m 255.255.255.0```

If ```-ip``` or ```-m``` doesn't exist, it will work exactly the same as before and won't break anything.

I've pushed it  to gateway master branch and didn't send a pull request

**Comment by scandox** [comment]
@liutingdu There is an issue with this. Certain cameras turn up no matter what range I give it. Since they're always Hikvision I suppose this is because onVif discovery ignores the range?

**Comment by liutingdu** [comment]
Yeah that must be the reason... I need to ignore the ONVIF device if the IP is not in that range. 

Thanks for pointing this out!

**Comment by scandox** [comment]
No probs! Actually my system works ok with it anyway as it ignores
duplicates

**Comment by liutingdu** [comment]
I pushed it to master again with the fixes, it must be working now.

Let me know if there are more issues!
